src/hooks/UseViewportSize/index.ts

```diff
@@ -0,0 +1,2 @@
+export {useViewportSize} from './UseViewportSize';
```

Comment:
Let's place this hook with others, inside utils directory, we will refactor directory structure later. Also the file should be named in lowercase

Reply:
fix